src/client/components/music/music.tsx

```diff
@@ -19,6 +19,11 @@ const MUSIC = [
 	"rbxassetid://9047105308", // Dusk To Dawn
 	"rbxassetid://9047105702", // Light Dreamer
 	"rbxassetid://9047105533", // No Smoking
+	"rbxassetid://1837441830", // Rattleback
+	"rbxassetid://1845743370", // Blue Night Waltz
```

Comment:
These two songs don't fit the overall vibe, but the other three work well!

Reply:
Just pushed a commit removing the songs and another adding 4 more. Hopefully they fit the mood better! Sorry for the extremely small PR, it's my first one.